### PR TITLE
[Backport][ipa-4-11] ipa-idrange-fix

### PR DESCRIPTION
**Pull request by KoRORland**
backport of ipa-idrange-fix to ipa-4-11

### CODE REVIEW

File: ipatests/prci_definitions/nightly_ipa-4-11_latest.yaml [hunk 1]
```diff
@@ -1794,6 +1794,19 @@ jobs:
...
         topology: *master_1repl
 
+  fedora-latest-ipa-4-11/test_idrange_fix:
+    requires: [fedora-latest-ipa-4-11/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-latest-ipa-4-11/build_url}'
+        selinux_enforcing: True
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-11_latest.yaml:1804]: Remove this line, the test should run in permissive mode

**Review by flo-renaud — COMMENTED** [review]
Hi @KoRORland 
thanks for the backport. Can you add a temp commit in order to trigger the new test?

**Comment by KoRORland** [comment]
added test commit, can't do re-run

**Comment by KoRORland** [comment]
For future ref, link to a successful run of the new test: [Details](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/2dcbe3cc-66d1-11ef-99c1-fa163e61f856)

**Comment by KoRORland** [comment]
selinux fixed, test commit removed

**Comment by flo-renaud** [comment]
ipa-4-11:

* 261e8eb56f9ecf2744b72a2683143b21ed230d11 Add ipa-idrange-fix
* 4cbfcba01e5aa544fed38c1e046c9cfc84d8fa2e Add PR-CI definitions

**Pull request closed without merging**